mix.exs

```diff
@@ -65,7 +65,9 @@ defmodule GSearcher.MixProject do
       {:ueberauth, "~> 0.6.3"},
       {:ueberauth_google, "~> 0.10.0"},
       {:oban, "~> 2.3"},
-      {:httpoison, "~> 1.7"}
+      {:httpoison, "~> 1.7"},
+      {:floki, "~> 0.29.0"},
+      {:exvcr, "~> 0.12.2"}
```

Comment:
Do we need exvcr as we can mock stuff by Mimic @bterone 🤔

Reply:
I think VCR might be better since Google's classes can change. In the future, it would be easier to re-record the VCRs than creating new mocks. What do you think? @andyduong1920 🤔

Reply:
Do we use ExVCR in this PR somewhere @bterone 🤔 , I can't see it :D

Reply:
Ah it use in `use_cassette`